iabgpp-encoder/src/main/java/com/iab/gpp/encoder/base64/TraditionalBase64UrlEncoder.java

```diff
@@ -0,0 +1,23 @@
+package com.iab.gpp.encoder.base64;
+
+public class TraditionalBase64UrlEncoder extends AbstractBase64UrlEncoder {
+
+  private static TraditionalBase64UrlEncoder instance = new TraditionalBase64UrlEncoder();
+  
+  private TraditionalBase64UrlEncoder() {
+    
+  }
+  
+  public static TraditionalBase64UrlEncoder getInstance() {
+    return instance;
+  }
+  
+  @Override
+  protected String pad(String bitString) {
+    while (bitString.length() % 24 > 0) {
```

Comment:
Worst-case, this could result in 23 copies of the string being created before the desired length is found. Could be done more efficiently by calculating the needed string first then appending.